Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN /usr/bin/fix-permissions /tmp/src \
     && /usr/bin/update-ca-trust
 RUN yum install -y krb5-workstation
 COPY data/kerberos/krb5.conf /etc
+RUN yum install -y skopeo
```

Comment:
Can we reuse the same `yum install` command from 2 line before this one?

Reply:
Sure, done